microsoft-365/security/office-365-security/message-trace-scc.md

```diff
@@ -58,7 +58,7 @@ The default values are **All senders** and **All recipients**, but you can use t
 - **To these people**: Click in this field to select one or more recipients in your organization.
 
 > [!NOTE]
-> You can also type the email addresses of external senders and recipients. Wildcards are supported (`*@contoso.com` or `scot?@contoso.com`), but you can't use multiple wildcard entries in the same field at the same time.<br/>You can paste multiple senders or recipients list separated with semicolon (`;`). Spaces (`\s`), carriage return (`\r`) or next lines (`\n`) symbols are allowed.
+> You can also type the email addresses of external senders and recipients. Wildcards are supported (for example, `*@contoso.com`), but you can't use multiple wildcard entries in the same field at the same time.<br/>You can paste multiple senders or recipients list separated with semicolon (`;`). Spaces (`\s`), carriage return (`\r`) or next lines (`\n`) symbols are allowed.
```

Comment:
```suggestion
> You can also type the email addresses of external senders and recipients. Wildcards are supported (for example, `*@contoso.com`), but you can't use multiple wildcard entries in the same field at the same time.<br/>You can paste multiple senders or recipients lists separated by semicolons (`;`). spaces (`\s`), carriage returns (`\r`), or next lines (`\n`).
```

Reply:
@MaratMussabekov - please merge the commit on the copy/edit review done by JohanFreelancer9 or make the necessary changes on the PR following the suggestions made. Thank you.